pkg/tournament/tournament.go

```diff
@@ -140,14 +140,14 @@ func SetTournamentMetadata(ctx context.Context, ts TournamentStore, meta *pb.Tou
 	}
 
 	if t.IsFinished {
-		return fmt.Errorf("tournament %s has finished", meta.Id)
+		return entity.NewWooglesError(realtime.WooglesError_TOURNAMENT_FINISHED, []string{t.Name, ""})
 	}
 
 	t.Lock()
 	defer t.Unlock()
 	name := strings.TrimSpace(meta.Name)
 	if name == "" {
-		return errors.New("name cannot be blank")
+		return entity.NewWooglesError(realtime.WooglesError_TOURNAMENT_EMPTY_NAME, []string{t.Name, ""})
```

Comment:
how come this and line 143 have an extra empty string?

Reply:
good question